### PR TITLE
Gruppe 3

### PR DESCRIPTION
**Pull request by DGCAS20**
Refactor Gruppe 3

### CODE REVIEW

File: src/main/java/de/cas/mse/exercise/SumDigPower.java [hunk 1]
```diff
@@ -5,37 +5,44 @@
...
+
+        for (int j = 0; j < currentNumberAsString.length(); j++) {
+        	digits.add(Long.valueOf(currentNumberAsString.substring(j, j + 1)));
+        }
+        return digits;
+    }
+
+    private long sumExponents(List<Long> digits) {
+        long sum = 0;
+        for (int power = 1; power <= digits.size(); power++) {
```
Comment (hpap) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:41]: maybe: index as iteration variable and power as index +1

File: src/main/java/de/cas/mse/exercise/SumDigPower.java [hunk 2]
```diff
@@ -5,37 +5,44 @@
 
 public class SumDigPower {
```
Comment (kufuntu) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:6]: consider renaming this as well

File: src/main/java/de/cas/mse/exercise/SumDigPower.java [hunk 3]
```diff
@@ -5,37 +5,44 @@
...
+			if (sum == currentNumber) {
+				eurekaNumbers.add(sum);
 			}
 
 		}
 
-        System.out.println(result);
-		return result;
-
+        System.out.println(eurekaNumbers);
```
Comment (kufuntu) [inline src/main/java/de/cas/mse/exercise/SumDigPower.java:24]: a get function should not have any side effects

File: src/main/java/de/cas/mse/exercise/SumDigPower.java (hunk 1 above)
Comment (temari) [reply src/main/java/de/cas/mse/exercise/SumDigPower.java:41]: and for the method name: maybe power would be better?

**Pull request merged**